src/store/api.js

```diff
@@ -4,6 +4,13 @@ const api = Axios.create({
   withCredentials: true
 });
 
+api.interceptors.response.use(undefined, error => {
```

Comment:
when we upstream, Could add a TODO with thoughts on how to handle this better

Reply:
I'm good without a TODO

Reply:
Do you have any suggestions? I am migrating this now.